integrations/access/pagerduty/app.go

```diff
@@ -312,10 +343,26 @@ func (a *App) getOnCallServiceNames(req types.AccessRequest) ([]string, error) {
 	return common.GetServiceNamesFromAnnotations(req, annotationKey)
 }
 
+func (a *App) getMessageRecipient(ctx context.Context, req types.AccessRequest) (string, error) {
```

Comment:
Nit: In the context of PagerDuty, "get message recipient" does not make much sense. Can we call this something else, or actually why not just update existing `getNotifyServiceName` method to return service name from the rule if it's present?